extensions/sql-database-projects/src/dialogs/createProjectFromDatabaseQuickpick.ts

```diff
@@ -0,0 +1,121 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as constants from '../common/constants';
+import { exists, getVscodeMssqlApi } from '../common/utils';
+import { IConnectionInfo } from 'vscode-mssql';
+import { defaultProjectNameFromDb, defaultProjectSaveLocation } from '../tools/newProjectTool';
+import { ImportDataModel } from '../models/api/import';
+import { mapExtractTargetEnum } from './createProjectFromDatabaseDialog';
+
+/**
+ * Create flow for a New Project using only VS Code-native APIs such as QuickPick
+ */
+export async function createNewProjectWithQuickpick(): Promise<ImportDataModel | undefined> {
```

Comment:
add "FromDb" or database to distinguish is from the other `createNewProjectWithQuickpick()` in the data-workspace extension